### DISCUSSION TITLE
2024-08-07 수요일

### DISCUSSION THREAD

**Discussion by Captain-Kim** [nbc-final-a10/TripBuddy#214] [기록보드]
2024-08-07 수요일 기록보드입니다.

## 여행가고십조(A10) 일일 공통 일과표

| 시간 | 일과 |  기록 |
| :-: | :-: | :-: |
| 09:00 ~ 10:00 | 아침 회의 | |
| 10:00 ~ 11:00 | 기술면접 준비 | |
| 11:00 ~ 13:00 | 집중 개발 time | |
| 13:00 ~ 14:00 | 취업 프로그램 세션 특강 | |
| 14:00 ~ 15:00 | 점심 식사 | |
| 14:00 ~ 18:00 | 집중 개발 time | |
| 18:00 ~ 19:00 | 저녁 식사 | |
| 19:00 ~ 20:00 | 저녁 회의 | |
| 20:00 ~ 21:00 | 5분 기록보드 작성 / 개인 회고 | |

```
## 개인 기록보드 서식 (김병준)

### 오늘 한 일

- 내용

### 트러블 슈팅

- 내용

### 내일 계획

- 내용
```

```
// 이미지 테이블
<table>
  <tr>
    <th>이미지 1번 제목</th>
    <th>이미지 2번 제목</th>
  </tr>
  <tr>
    <td><img src="이미지 주소" width="300" height="300"/></td>
    <td><img src="이미지 주소" width="300" height="300"/></td>
  </tr>
</table>
```

**Comment by Captain-Kim** [comment]
## 개인 기록보드 서식 (김병준)

### 오늘 한 일

- 팔로우 기능 개발 (팔로우 API 라우트 구성, 중복 검사, 팔로우, 언팔로우)
- 버디 프로필 tanstack query 리팩토링

### 내일 계획

- 리치텍스트에디터 구현 시작

**Comment by slsl2** [comment]
## 개인 기록보드 서식 (김민지)

### 오늘 한 일

- 채팅 목록 프로필 사진 위치 margin 값으로 조정
- 채팅 목록 프로필 사진이 fetch 안 되면 회색 동그라미가 뜨도록 UX 개선
- 전체적으로 스타일링 수정 (디자인 최종본으로)
